### PR TITLE
Some small corrections. Lang file and event deprecated message

### PR DESCRIPTION
**Pull request by nmoller**
Some error messages caused by langfile.
A deprecation warning was cleared.

### CODE REVIEW

**Comment by mgage** [comment]
Thanks.  These modules have been used in some form by a number of WeBWorK community members, usually as part of the wwmoodle package on the github.com/openwebwork site.  This is my attempt to start cleaning these up so that they can be more easily used by those used to moodle, but perhaps not used to WeBWorK.  

Your contributions are appreciated. 
-- Mike

File: view.php [hunk 1]
```diff
@@ -104,9 +104,9 @@
       . "<a href='$wwsetlink'>" . get_string("iframeNoShow-2", "wwassignment")
       ."</a><p align='center'></iframe></p>\n"
       );
-print("<iframe id='wwPage' src='$wwsetlink' frameborder='1' "
+print("<iframe id='wwPage' src='$wwsetlink' frameborder='0' "
       . "width='".$CFG->wwassignment_iframewidth."' "
-      . "height='".$CFG->wwassignment_iframeheight."'>"
+      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"
```
Comment (mgage) [inline view.php:109]: On second thought this change probably a mistake.  There are many webwork problems which don't fit within a frame.  Without scrolling the bottom of the problem (with the submit button and other information) can't be displayed.

Comment (nmoller) [inline view.php:109]: I've done a quick review of the responsive capacities of putting everything without a scroll and everything seemed to fit. But I should be further tested...

Comment (mgage) [inline view.php:109]: https://devel3.webwork.rochester.edu/moodle/mod/wwassignment/view.php?id=1022  
(Linear Algebra and ODE, Spring 2009 -- the homework set is WebWork Orientation -- problem 1
(on this site I have reset the iframe to allow scrolling).    

There are also other problems, "scaffolding problems" which can be very long  https://hosted2.webwork.rochester.edu/webwork2/2014_HKUST_demo/SequentialWordProblem/1/.

Finally whether scrolling is needed or not may depend a great deal on what device you are using to view the problem.  I think it is safer to allow scrolling.  I don't see any particular downside to having the option available.

Comment (nmoller) [inline view.php:109]: Thanks Michael for the links!
I was working on the wwassignment module... to get it properly I should do an upgrade of the table:
add field introformat and rename description into intro, if you agree with that change the mod_form should be updated to use the intro instead...
You can look at the module of your choice to see those fields (ex mod_label : https://github.com/moodle/moodle/blob/master/mod/label/db/install.xml
)
Do you agree if I do that?

Comment (mgage) [inline view.php:109]: I think so.  I'm on the road right now going to my fathers house in canton ny.  I'll look more closely when I get there.  I appreciate the help in bringing these modules up to the current best practices for moodle modules

Ttyl

Sent from my iPhone

On Mar 9, 2015, at 12:12 PM, Nelson Moller <notifications@github.com<mailto:notifications@github.com>> wrote:

In view.phphttps://github.com/mgage/wwassignment/pull/1#discussion_r26049273:

> ```
>    . "width='".$CFG->wwassignment_iframewidth."' "
> ```
> -      . "height='".$CFG->wwassignment_iframeheight."'>"
> -      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"

Thanks Michael for the links!
I was working on the wwassignment module... to get it properly I should do an upgrade of the table:
add field introformat and rename description into intro, if you agree with that change the mod_form should be updated to use the intro instead...
You can look at the module of your choice to see those fields (ex mod_label : https://github.com/moodle/moodle/blob/master/mod/label/db/install.xml
)
Do you agree if I do that?

##

**Comment by nmoller** [comment]
Dont worry this is not a hurry :)... Have a nice trip to your parents place.

You can see the work in progress at https://github.com/nmoller/wwassignment
One question, I imagine that most people added the description as html... if that is the case I should change the upgrade method to take that into consideration.

**Pull request merged**